Examples/Algorithms/Digitization/include/ActsExamples/Digitization/DigitizationAlgorithm.hpp

```diff
@@ -11,6 +11,7 @@
 #include "Acts/Definitions/Algebra.hpp"
 #include "Acts/Definitions/TrackParametrization.hpp"
 #include "Acts/Geometry/GeometryHierarchyMap.hpp"
+#include "ActsExamples/Digitization/AlgorithmConfig.hpp"
 #include "ActsExamples/Digitization/DigitizationConfig.hpp"
```

Comment:
We should probably find a different name for the `DigitizationConfig` and use that for the algorithm configuration, what do you think?

Reply:
I agree, there is a risk that someone writes "using namespace Digitization" and then we end up with a bare `AlgorithmConfig` that seems way more generic than it is. But I would not change the DigitizationConfig class to GeometricDigitizationConfig since in this PR this class is re-used by the SmearingAlgorithm. Maybe `DigitizationParameters`? I'll give it some more thought.

Reply:
There is already a type named `DigitizedParameters` which may be confusing if we take up my previous suggestion

Reply:
Some more ideas to replace DigitizationConfig: SpecificDigiConfig, DigiComponentsConfig. I like the second one since it accurately describes the struct wrt DigitizationAlgorithm, which has two components (Geo and Smear)

Reply:
I went with DigiComponentsConfig. Let me know if this doesn't work @asalzburger